.github/workflows/Build.yml

```diff
@@ -13,6 +13,17 @@ env:
   kphp_build_dir: /home/kitten/kphp/build
 
 jobs:
+  # clang-format:
```

Comment:
You don't need that code? Why don't you delete it?

Reply:
Oh. As I understand it is for clang-format check. Let's turn on it

Reply:
@astrophysik, yes, it should definitely be enabled. But let's do it in another MR.

To do it right, you have to at least do the preparation to make it work the same locally and on CI :)

Reply:
removed